samples/net/updatehub/prj.conf

```diff
@@ -2,7 +2,7 @@
 CONFIG_UPDATEHUB=y
```

Comment:
Could you split the sample changes in commit `lib: updatehub: Fix buffer sizes` to different commit as they are kind of separate changes.

Reply:
Just a double check before split. Since we fix buffer values the system don't build with _CONFIG_HEAP_MEM_POOL_SIZE=2048_. In fact, this was the motivation to increase that value.
Should I split anyway?

Reply:
Ah, if they do not build without the change, then leave it like it is now.